swatch-tally/deploy/clowdapp.yaml

```diff
@@ -49,6 +49,9 @@ parameters:
     value: 1500m
   - name: MIGRATION_IMAGE
     value: quay.io/cloudservices/swatch-database
+  - name: MIGRATION_IMAGE_TAG
+    description: The tag of the migration image to use (only for component-testing purposes)
+    value: ${IMAGE_TAG}
```

Comment:
Unfortunately, this won't work because it's not supported by OpenShift :(

Reply:
There are no workarounds. What we'll do for the swatch-contracts service (and swatch-billable-usage service in the future) is to move the liquibase scripts from swatch-database to this service back and do not depend on swatch-database anylonger.

Reply:
Sounds good. I'll remove the clowdapp changes and merge this and create some follow up cards. Thanks for your help with this!